test/integration/puppetlabs/services/jruby/jruby_puppet_pool_int_test.clj

```diff
@@ -149,7 +149,7 @@
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 ;;; Tests
 
-(deftest ^:integration ^:single-threaded-only admin-api-flush-jruby-pool-test
```

Comment:
This test really shouldn't pass... if it does, that's coincidental, since the assertions don't make sense for a pool with one instance. I still think it should have a different version written for the multithreaded pool that demonstrates we're only dealing with a single instance.

Reply:
oh yeah. I removed this when I saw things pass for me locally earlier this week, but I don't think I actually ran in multithreaded. I can put it back.

If there isn't a ticket for the tests that need addressing, can you open one?

Reply:
Yes.

Reply:
Thank you!!

Reply:
https://tickets.puppetlabs.com/browse/SERVER-2721